File: persistence/src/main/java/com/example/diploma/persistence/dto/admin/UserDto.java
package com.example.diploma.persistence.dto.admin;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class UserDto {

        Long id;
        String fullname;
        String phoneNumber;
        Boolean isActive;
        String city;
        String country;
        String username;
        List<String> authorities;

}

File: admin/src/main/frontend/src/app/dto/UserDto.ts
export class UserDto {

	id: number;
	fullname: string;
	phoneNumber: string;
	isActive: boolean;
	city: string;
	country: string;
	email: string;


	constructor(id?: number, fullname?: string, phoneNumber?: string, isActive?: boolean,
							city?: string, country?: string, email?: string) {
		this.id = id;
		this.fullname = fullname;
		this.phoneNumber = phoneNumber;
		this.isActive = isActive;
		this.city = city;
		this.country = country;
		this.email = email;
	}
}